src/backend/web/graphql/index.js

```diff
@@ -26,15 +26,14 @@ module.exports.typeDefs = graphql`
   # 'Post' matches our Post type used with redis
   type Post {
     id: String
-    author: String
     title: String
     html: String
     text: String
     published: String
     updated: String
     url: String
-    site: String
     guid: String
+    feed: Feed!
```

Comment:
Why is it Feed!  and not Feed

(Curiosity)

Reply:
@c3ho was telling me it means "this field is required"